scripts/schemaGenerator.py

```diff
@@ -55,6 +58,36 @@ def get_options_list_for_enum(field):
     return options_list
 
 
+def is_dynamic_config_supported_field(field, dbConfig=None):
+    if dbConfig is None or "dynamicConfigSupported" not in dbConfig:
+        # Case where dbConfig doesn't contain dynamicConfigSupported field at all
+        return False
+    dynamicConfigSupported = dbConfig["dynamicConfigSupported"]
+    configKey = ""
+    if "configKey" in field:
+        configKey = field["configKey"]
+    isDynamicConfigSupportedField = configKey in dynamicConfigSupported
+    return isDynamicConfigSupportedField
+
+
+def generate_uiconfig_pattern(field, dbConfig=None) -> str:
+    field_supports_dynamic_config = is_dynamic_config_supported_field(field, dbConfig)
+    destName = ""
+    if "name" in dbConfig:
```

Comment:
do we need this check ? `name` is mandatory field in dbConfig ? If it's missing, rather script should fail instead of handling here

Reply:
`dbConfig` doesn't indicate `db-config.json` but it includes `db-config-json.config`. And the config doesn't contain `name` natively. I had to manually include it in `schemaGenerator.py`

Reply:
The check has been added to make sure tests are passing 🙁

Reply:
since you added `name` to dbConfig["name"] now, so it will definitely be present as `name` is mandatory in all definitions. Due to any bug/issue, if destName doesn't get populated, then `generate_uiconfig_pattern` might not return intended response.
if you still want to have the check, do add error logs if destName is missing - gets captured in github action logs

Reply:
> since you added name to dbConfig["name"] now, so it will definitely be present as name is mandatory in all definitions

The name I think was not added in some test-cases, which was throwing error. The condition would not cause exceptions. I guess it is fine to have the logic as is.

Reply:
`name` is not needed now.